packages/bpk-react-utils/src/Portal.js

```diff
@@ -147,7 +147,7 @@ class Portal extends Component {
       return this.props.renderTarget();
     }
 
-    return this.props.renderTarget || document.body;
```

Comment:
@matthewdavidson we're now accepting a function as render target so this bit can be removed right?

Reply:
Sure 👍